docs/platforms/cwp/index.md

```diff
@@ -1,5 +1,74 @@
 [](){#ref-platform-cwp}
-# Climate and Weather Platform
+# Climate and weather platform
+
+The Machine Learning Platform (MLp) provides compute, storage and support to the climate and weather modeling community in Switzerland.
+
+## Getting Started
+
+### Getting access
+
+Project administrators (PIs and deputy PIs) of projects on the MLp can to invite users to join their project, before they can use the project's resources on Alps.
 
 !!! todo
-    follow the template of the [MLp][ref-platform-mlp]
+    This points to the Waldur solution - whether the [UMP][ref-account-ump] or [Waldur][ref-account-waldur] docs are linked depends on which is being used when these docs go live.
+
+This is performed using the [project management tool][ref-account-waldur]
+
+Once invited to a project, you will receive an email, which you can need to create an account and configure [multi-factor authentification][ref-mfa] (MFA).
+
+## Systems
+
+Santis is the system deployed on the Alps infrastructure for the climate and weather platform.
+Its name derives from the highest mountain Säntis in the Alpstein massif of North-Eastern Switzerland.
+
+<div class="grid cards" markdown>
+-   :fontawesome-solid-mountain: [__Santis__][ref-cluster-santis]
+
+    Santis is a large [Grace-Hopper][ref-alps-gh200-node] cluster.
+</div>
+
+[](){#ref-cwp-storage}
+## File systems and storage
+
+There are three main file systems mounted on the CWp system Santis.
+
+| type |mount | filesystem |
+| -- | -- | -- |
+| Home | /users/$USER | [VAST][ref-alps-vast] |
+| Scratch | `/capstor/scratch/cscs/$USER` | [Iopstor][ref-alps-capstor] |
```

Comment:
```suggestion
| Scratch | `/capstor/scratch/cscs/$USER` | [Iopstor][ref-alps-iopstor] |
```

Reply:
Sorry, the change should have been `[Iopstor]` to `[Capstor]`